src/Sulu/Bundle/WebsiteBundle/Tests/Unit/EventListener/AppendAnalyticsListenerTest.php

```diff
@@ -57,7 +57,7 @@ public function testAppendFormatNoEffect($format): void
         $request = $this->prophesize(Request::class);
         $request->getRequestFormat()->willReturn($format);
         $response = $this->prophesize(Response::class);
-        $response->reveal()->headers = new ParameterBag(['Content-Type' => 'text/plain']);
+        $response->reveal()->headers = new HeaderBag(['Content-Type' => 'text/plain']);
```

Comment:
@mamazu looks like there are 2 headerbags one for request and one for response here we require the `ResponseHeaderBag`